high_card.rb

```diff
@@ -3,3 +3,79 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+DEBUG = true
+
+TIE_MARKER = "# " * 5
+ROUND_MARKER = "\n" + "* " * 25 + "\n\n"
+
+deck = []
+
+suits.each do |suit|
+  ranks.each do |rank|
+    deck.push([rank, suit])
+  end
+end
```

Comment:
may be better to use a hash to symbolize cards. Instead of indexing by cryptic integer values (can be at times at least) might be better to index by semantic symbols.